### ISSUE TITLE
Error since update

### ISSUE THREAD

**Issue by sucrecacao** [alexherbo2/snippets.kak#9]
I was working well but now I got the following error:
`Error: 1:2: 'snippets-menu' 2:5: 'evaluate-commands' no such command: 'snippets-python-menu'
`

No idea where it come from. I did hack with my kakrc a bit but nothing that should impact the pluggin.

**Comment by alexherbo2** [comment]
What do you have in your `*debug*` buffer?

**Comment by sucrecacao** [comment]
```
*** This is the debug buffer, where debug info will be written ***
Error: 1:2: 'snippets-menu' 2:5: 'evaluate-commands' no such command: 'snippets-python-menu'
```

**Comment by sucrecacao** [comment]
Fixed but its weird..
I created a bash script named `kak` with the following `PATH=.config/kak/bin kak` ( it just add a bin path when running kakoune for the following reason https://discuss.kakoune.com/t/having-a-config-kak-bin-just-for-kakoune-script-and-binary/1190 )  .

But he full script should be ``PATH=.config/kak/bin kak $@` . I forgot to put a `$@` at the end to pass the argument.  

This broke snippets apparently. It work again since I added the `$@`.
